quetz/main.py

```diff
@@ -47,6 +47,9 @@ def get_db():
     db = SessionLocal()
     try:
         yield db
+    except IntegrityError:
+        db.rollback()
```

Comment:
This ultimately happens implicitly by way of calling close() in the finally block. However I'll remove the top commit for now until the best way to handle this is decided.

I agree we should have the database creation run before the package is committed to disk, though I don't think we should "look before we leap", since that will be prone to race conditions.